src/lib/components/ratio/ColorIssues.svelte

```diff
@@ -29,10 +29,21 @@
 </details>
 
 <style lang="scss">
-  dl {
-    display: block;
+  .known-issues {
+    grid-area: knownissues;
+    padding: var(--shim) var(--double-gutter);
```

Comment:
@stacyk This spacing was a little tricky. When the colors are passing this section seems sort of misaligned. 

![image](https://github.com/oddbird/oddcontrast/assets/41588129/7a5b3b79-1651-4f2c-b521-80edb1103381)

But that padding is needed once the colors fail and the background is added

![image](https://github.com/oddbird/oddcontrast/assets/41588129/13db30ba-e3b1-425c-b39f-116a5d770d17)

Reply:
@stacyk This worked to get the text aligned correctly, but that made the marker end up in a weird place. I took most of your suggestions above but also added a custom marker to the other side of the summary text:

![image](https://github.com/oddbird/oddcontrast/assets/41588129/6b117c4c-d707-4301-8646-75e828ed9948)

![image](https://github.com/oddbird/oddcontrast/assets/41588129/3f5f3423-666c-4db2-9e8e-77554f1f292c)

What do you (and @Sondra) think of this placement?